.travis/downloadCairo.sh

```diff
@@ -0,0 +1,4 @@
+
+curl -L https://github.com/preshing/cairo-windows/releases/download/$CAIRO_VERSION/cairo-windows-$CAIRO_VERSION.zip -o cairocomplied.zip
```

Comment:
This is an outdated cairo release :(

Reply:
The only thing we can do now is to compile Cairo if we need the latest one. And btw the install script in [cairo-windows](https://github.com/preshing/cairo-windows) is actually batch which Travis doesn't support :(

Reply:
yeah. It's probably best to wait until https://gitlab.freedesktop.org/cairo/cairo/-/merge_requests/37 is merged and then build it ourselves using meson. But, yeah, let's not block this..

Reply:
> yeah. It's probably best to wait until https://gitlab.freedesktop.org/cairo/cairo/-/merge_requests/37 is merged and then build it ourselves using meson. But, yeah, let's not block this..

FYI: It's been merged.

Reply:
Is the idea that we would use meson to do the builds here, or is there some buildfarm attached to freedesk that we get the builds from ?

OT (ish)
About a week ago, I had a chat with dependencies.io and they came up with a patch that uses their bot to generate PRs to update cairo + freedesktop, for preshing, cairo when upstream changes.   This isn't so relevant if we don't use preshing cairo, but I guess we can still setup something to generate PRs here when upstream updates.

Reply:
SO what should be done?